docs/gemini/gemini.rst

```diff
@@ -0,0 +1,123 @@
+.. doctest-skip-all
+
+.. _astroquery.gemini:
+
+************************************
+Gemini Queries (`astroquery.gemini`)
+************************************
+
+Getting Started
+===============
+
+This module can be used to query the Gemini Archive. Below are examples of querying the data with various
+parameters.
+
+Positional Queries
+------------------
+
+Positional queries can be based on a sky position.  Radius is an optional parameter and the default is 0.3 degrees.
+
+.. code-block:: python
+
+                >>> from astroquery.gemini import Observations
+                >>> from astropy import coordinates, units
+
+                >>> coord = coordinates.SkyCoord(210.80242917, 54.34875, unit="deg")
+                >>> data = Observations.query_region(coordinates=coord, radius=0.3*units.deg)
+                >>> print(data[0:5])
+                
+                exposure_time detector_roi_setting detector_welldepth_setting  telescope   ...
+                ------------- -------------------- -------------------------- ------------ ...
+                     119.9986           Full Frame                         -- Gemini-North ...
+                     119.9983           Full Frame                         -- Gemini-North ...
+                     119.9986           Full Frame                         -- Gemini-North ...
+                     119.9983           Full Frame                         -- Gemini-North ...
+                      99.9983           Full Frame                         -- Gemini-North ...
+
+
+Observation Name Queries
+------------------------
+
+You may also do a query by the name of the object you are interested in.
+
+.. code-block:: python
+
+                >>> from astroquery.gemini import Observations
+
+                >>> data = Observations.query_object(objectname='m101')
+                >>> print(data[0:5])
+
+                exposure_time detector_roi_setting detector_welldepth_setting  telescope   ...
+                ------------- -------------------- -------------------------- ------------ ...
+                     119.9986           Full Frame                         -- Gemini-North ...
+                     119.9983           Full Frame                         -- Gemini-North ...
+                     119.9986           Full Frame                         -- Gemini-North ...
+                     119.9983           Full Frame                         -- Gemini-North ...
+                      99.9983           Full Frame                         -- Gemini-North ...
+
+
+Observation Criteria Queries
+----------------------------
+
+Additional search terms are available as optional arguments to the `~astroquery.gemini.ObservationsClass.query_criteria`
+call.  These all have default values of None, in which case they will not be considered during the search.
+
+The full API is in the api documentation, but some examples are the instrument used, such as GMOS-N, the
```

Comment:
link to the API documentation when you mention it

Reply:
hmm, I meant the reference from the earlier line but that isn't clear.  I've tried rewording it a bit.